doc/mgmt/Management Framework.md

```diff
@@ -2196,6 +2358,27 @@ Above is the sequence diagram explaining the CVL steps. Note that interaction be
 25. Write lock acquired in Step 3 is released.
 26. Final response is returned from the Translib infra to REST/GNMI.
 
+### 4.3 NETCONF <get> flow
```

Comment:
The section should be 4.6
![image](https://github.com/user-attachments/assets/248e903f-5427-4a2e-8ee6-730f9e7bf941)